modules/global-config/src/main/java/org/mule/runtime/globalconfig/internal/MavenConfigBuilder.java

```diff
@@ -106,6 +109,10 @@ public static MavenConfiguration buildMavenConfig(Config mavenConfig) {
               if (password != null) {
                 authenticationBuilder.password(password);
               }
+              getRepositoryPolicy(repositoryConfig, "snapshotPolicy")
+                  .ifPresent(snapshotPolicy -> remoteRepositoryBuilder.snapshotPolicy(snapshotPolicy));
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this lambda with a method reference. ([squid:S1612](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1612))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7660&issues=AWjOEnc1otIXnmsUhvjM&open=AWjOEnc1otIXnmsUhvjM)

[ISSUE_KEY]: AWjOEnc1otIXnmsUhvjM